code/datums/components/crafting/crafting.dm

```diff
@@ -458,6 +478,15 @@
 	tool_text = replacetext(tool_text,",","",-1)
 	data["tool_text"] = tool_text
 
+	for(var/path_or_instance in R.machinery)
+		if(ispath(path_or_instance, /obj/machinery))
+			var/obj/machinery/path = path_or_instance
+			machinery_text += " [initial(path.name)],"
+		else
+			machinery_text += " [path_or_instance],"
```

Comment:
```suggestion
			machinery_text += " [initial(path.name)]"
		else
			machinery_text += " [path_or_instance.name]"
```
* Commas removed, added in the `Join()` glue arg.
* `name` is what is defaulted to, but in this case we are making explicit for clarity.